GPU/GX2/ShaderManagerGX2.h

```diff
@@ -56,8 +56,13 @@ enum class UB_Bindings : u32 {
 
 } // namespace GX2Gen
 
-struct UB_VSID
-{
+struct UB_VSID {
+	struct {
+		u32_le COMP;
```

Comment:
Might be able to pack a lot of this stuff into bitfields, if you care about saving uniform buffer space..